### PR TITLE
Repocleanliness

### PR DESCRIPTION
**Pull request by DawidSzpener**
I've changed the model field names to lowercase, to reflect the python syntax convention and updated all the related files (test including).

### CODE REVIEW

File: static/static_csv/invalidasset.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-AssetTag,DeviceType,DeviceModel,AssetStatus,SerialNumber,AssetCondition,ScreenSize,HardDrive,Ram,Year
+asset_tag,device_type,device_model,asset_status,serial_number,asset_condition,screen_size,hard_drive,ram,year
```
Comment (IliasMariosG) [inline static/static_csv/invalidasset.txt:1]: add spaces after commas

File: static/static_csv/invalidasset.csv [hunk 2]
```diff
@@ -1,4 +1,4 @@
-AssetTag,DeviceType,DeviceModel,AssetStatus,SerialNumber,AssetCondition,ScreenSize,HardDrive,Ram,Year
+asset_tag,device_type,device_model,asset_status,serial_number,asset_condition,screen_size,hard_drive,ram,year
```
Comment (IliasMariosG) [inline static/static_csv/invalidasset.csv:1]: add one space after each comma

File: static/static_csv/exampleasset.csv [hunk 3]
```diff
@@ -1,4 +1,4 @@
-AssetTag,DeviceType,DeviceModel,AssetStatus,SerialNumber,AssetCondition,ScreenSize,HardDrive,Ram,Year
+asset_tag,device_type,device_model,asset_status,serial_number,asset_condition,screen_size,hard_drive,ram,year
```
Comment (IliasMariosG) [inline static/static_csv/exampleasset.csv:1]: Add space after each comma

File: static/static_csv/Template.csv [hunk 4]
```diff
@@ -1 +1 @@
-AssetTag,DeviceType,AssetStatus,SerialNumber
\ No newline at end of file
+asset_tag,device_type,device_model,asset_status,serial_number,asset_condition,screen_size,hard_drive,ram,year
```
Comment (IliasMariosG) [inline static/static_csv/Template.csv:1]: Add space after each comma

File: AssetTracker/apps/register/models.py [hunk 5]
```diff
@@ -1,9 +1,9 @@
...
 from django.contrib.auth.models import User
-from companies.models import Company
+from ..companies.models import Company
 from django.contrib.auth.hashers import make_password
 
 class UnauthorizedUser(models.Model):
-  User = models.ForeignKey(User, on_delete=models.SET_NULL, null=True)
-  Company = models.ForeignKey(Company, on_delete=models.CASCADE, null=True)
+  user = models.ForeignKey(User, on_delete=models.SET_NULL, null=True)
+  company = models.ForeignKey(Company, on_delete=models.CASCADE, null=True)
```
Comment (IliasMariosG) [inline AssetTracker/apps/register/models.py:9]: To add newline to the end of the file

File: AssetTracker/apps/importcsv/forms.py [hunk 6]
```diff
@@ -0,0 +1,7 @@
+from django import forms
+from ..assets.models import Asset
+
+class AssetForm(forms.ModelForm):
+  class Meta:
+    model = Asset
+    fields = ("asset_tag", "device_type", "device_model", "asset_status", "serial_number", "asset_condition", "screen_size", "hard_drive", "ram", "year", "created_by", "company")
```
Comment (IliasMariosG) [inline AssetTracker/apps/importcsv/forms.py:7]: Maybe the use of single and double quotes, but not quite sure. We can chat about it on the reetro

File: AssetTracker/apps/functionaltests/test_dashboard.py [hunk 7]
```diff
@@ -5,20 +5,20 @@
...
+from ..companies.models import Company
+from ..companyusers.models import CompanyUser
 import time
 
 class Dashboard(LiveServerTestCase):
 
   def setUp(self):
     self.browser = webdriver.Firefox()
-    self.company = Company(Name="Makers")
+    self.company = Company(name="Makers")
```
Comment (IliasMariosG) [inline AssetTracker/apps/functionaltests/test_dashboard.py:17]: The use of single and double quotes

File: AssetTracker/apps/functionaltests/test_asset.py [hunk 8]
```diff
@@ -5,29 +5,29 @@
...
     self.company2.save()
     self.user2 = User.objects.create_user(username='admin2', password='admin2', email='test2@test.com', is_active=True)
     self.user2.save()
-    self.company_user2 = CompanyUser.objects.create(User=self.user2, Company=self.company2)
+    self.company_user2 = CompanyUser.objects.create(user=self.user2, company=self.company2)
     self.user3 = User.objects.create_user(username='admin3', password='admin3', email='test3@test.com', is_active=True)
     self.user3.save()
-    self.company_user3 = CompanyUser.objects.create(User=self.user3, Company=self.company)
-    self.A = Asset(AssetTag='BR20RL', DeviceType='Laptop', AssetStatus='Ready', SerialNumber='5', CreatedBy=self.user, Company=self.company)
+    self.company_user3 = CompanyUser.objects.create(user=self.user3, company=self.company)
```
Comment (IliasMariosG) [inline AssetTracker/apps/functionaltests/test_asset.py:29]: Space before and after '=' signs

File: AssetTracker/apps/functionaltests/test_asset.py [hunk 9]
```diff
@@ -5,29 +5,29 @@
...
+    self.A = Asset(asset_tag='BR20RL', device_type='Laptop', asset_status='Ready', serial_number='6', created_by=self.user, company=self.company)
```
Comment (IliasMariosG) [inline AssetTracker/apps/functionaltests/test_asset.py:30]: Space before and after '=' signs

File: AssetTracker/apps/functionaltests/test_asset.py [hunk 10]
```diff
@@ -5,29 +5,29 @@
...
+from ..companies.models import Company
+from ..companyusers.models import CompanyUser
 import time
 
 class AssetTest(LiveServerTestCase):
 
   def setUp(self):
     self.browser = webdriver.Firefox()
-    self.company = Company(Name="Makers")
+    self.company = Company(name="Makers")
```
Comment (IliasMariosG) [inline AssetTracker/apps/functionaltests/test_asset.py:17]: Space before and after '=' signs

File: AssetTracker/apps/functionaltests/test_asset.py [hunk 11]
```diff
@@ -5,29 +5,29 @@
...
     self.company.save()
     self.user = User.objects.create_user(username='admin1', password='admin1', email='test@test.com', is_active=True)
     self.user.save()
-    self.company_user = CompanyUser.objects.create(User=self.user, Company=self.company)
-    self.company2 = Company(Name="IBM")
+    self.company_user = CompanyUser.objects.create(user=self.user, company=self.company)
```
Comment (IliasMariosG) [inline AssetTracker/apps/functionaltests/test_asset.py:21]: Space before and after '=' signs

Comment (IliasMariosG) [inline AssetTracker/apps/functionaltests/test_asset.py:21]: Space before and after '=' signs

File: AssetTracker/apps/functionaltests/test_asset.py [hunk 12]
```diff
@@ -5,29 +5,29 @@
...
+    self.company2 = Company(name="IBM")
     self.company2.save()
     self.user2 = User.objects.create_user(username='admin2', password='admin2', email='test2@test.com', is_active=True)
     self.user2.save()
-    self.company_user2 = CompanyUser.objects.create(User=self.user2, Company=self.company2)
+    self.company_user2 = CompanyUser.objects.create(user=self.user2, company=self.company2)
```
Comment (IliasMariosG) [inline AssetTracker/apps/functionaltests/test_asset.py:26]: Space before and after '=' signs

File: AssetTracker/apps/functionaltests/test_asset.py [hunk 13]
```diff
@@ -5,29 +5,29 @@
...
     self.browser = webdriver.Firefox()
-    self.company = Company(Name="Makers")
+    self.company = Company(name="Makers")
     self.company.save()
     self.user = User.objects.create_user(username='admin1', password='admin1', email='test@test.com', is_active=True)
     self.user.save()
-    self.company_user = CompanyUser.objects.create(User=self.user, Company=self.company)
-    self.company2 = Company(Name="IBM")
+    self.company_user = CompanyUser.objects.create(user=self.user, company=self.company)
+    self.company2 = Company(name="IBM")
```
Comment (IliasMariosG) [inline AssetTracker/apps/functionaltests/test_asset.py:22]: Space before and after '=' signs

File: AssetTracker/apps/companyusers/serializers.py [hunk 14]
```diff
@@ -4,4 +4,4 @@
 class CompanyUserSerializer(serializers.ModelSerializer):
     class Meta:
         model = CompanyUser
-        fields = ('User', 'Company')
\ No newline at end of file
+        fields = ('user', 'company')
```
Comment (IliasMariosG) [inline AssetTracker/apps/companyusers/serializers.py:7]: To add newline to the end of the file

File: AssetTracker/apps/companyusers/migrations/0001_initial.py [hunk 15]
```diff
@@ -18,8 +18,8 @@ class Migration(migrations.Migration):
         migrations.CreateModel(
             name='CompanyUser',
             fields=[
-                ('User', models.OneToOneField(on_delete=django.db.models.deletion.CASCADE, primary_key=True, serialize=False, to=settings.AUTH_USER_MODEL)),
-                ('Company', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='companies.Company')),
+                ('user', models.OneToOneField(on_delete=django.db.models.deletion.CASCADE, primary_key=True, serialize=False, to=settings.AUTH_USER_MODEL)),
```
Comment (IliasMariosG) [inline AssetTracker/apps/companyusers/migrations/0001_initial.py:21]: To add space before and after each '=' sign wherever required

File: AssetTracker/apps/companyusers/migrations/0001_initial.py [hunk 16]
```diff
@@ -18,8 +18,8 @@ class Migration(migrations.Migration):
...
+                ('company', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='companies.Company')),
```
Comment (IliasMariosG) [inline AssetTracker/apps/companyusers/migrations/0001_initial.py:22]: To add space before and after each '=' signs

File: AssetTracker/apps/companies/models.py [hunk 17]
```diff
@@ -2,5 +2,5 @@
 from django.contrib.auth.models import User
 # Create your models here.
 class Company(models.Model):
-  Name = models.CharField(max_length=30, unique=True)
-  Owned_by = models.ForeignKey(User, on_delete=models.CASCADE, null=True)
\ No newline at end of file
+  name = models.CharField(max_length=30, unique=True)
+  owned_by = models.ForeignKey(User, on_delete=models.CASCADE, null=True)
```
Comment (IliasMariosG) [inline AssetTracker/apps/companies/models.py:6]: To add newline to the end the file

**Review by IliasMariosG — APPROVED** [review]
I can resolve the comments made in the `pylintConfiguration` branch

File: AssetTracker/apps/importcsv/forms.py (hunk 6 above)
Comment (DawidSzpener) [reply AssetTracker/apps/importcsv/forms.py:7]: Let's leave doubles for now, I look up what is the convention and it's hard to figure out actually

File: AssetTracker/apps/companyusers/migrations/0001_initial.py (hunk 15 above)
Comment (DawidSzpener) [reply AssetTracker/apps/companyusers/migrations/0001_initial.py:21]: This one is automatically generated file when you make migrations

File: AssetTracker/apps/functionaltests/test_asset.py (hunk 8 above)
Comment (DawidSzpener) [reply AssetTracker/apps/functionaltests/test_asset.py:29]: I am a bit skeptical about this, I tried and it looks much worse 🤣

File: AssetTracker/apps/companyusers/serializers.py (hunk 14 above)
Comment (DawidSzpener) [reply AssetTracker/apps/companyusers/serializers.py:7]: > I can resolve the comments by working on the `pylintConfiguration` branch.
> There might be a few things I missed, but I did my best.

All good my friend, fixed everything you mentioned but I am not gonna touch the spaces around =, it really looks weird. Thanks for taking the time to review, great stuff.

**Comment by DawidSzpener** [comment]
> I can resolve the comments by working on the `pylintConfiguration` branch.
> There might be a few things I missed, but I did my best.

All good my friend, fixed everything you mentioned but I am not gonna touch the spaces around =, it really looks weird. Thanks for taking the time to review, great stuff.

**Review by IliasMariosG — APPROVED** [review]

**Comment by IliasMariosG** [comment]
> > I can resolve the comments by working on the `pylintConfiguration` branch.
> > There might be a few things I missed, but I did my best.
> 
> All good my friend, fixed everything you mentioned but I am not gonna touch the spaces around =, it really looks weird. Thanks for taking the time to review, great stuff.

Thanks, Dawid.

**Pull request merged**